PhysicsTools/PatAlgos/python/triggerLayer1/triggerProducer_cfi.py

```diff
@@ -7,7 +7,7 @@
 ,l1GtTriggerMenuLiteInputTag = cms.InputTag("gtDigis")
 ,l1tAlgBlkInputTag = cms.InputTag("gtStage2Digis")
 ,l1tExtBlkInputTag = cms.InputTag("gtStage2Digis")
-,ReadPrescalesFromFile = cms.bool(True)
+,ReadPrescalesFromFile = cms.bool(False)
```

Comment:
@rekovic 
please clarify why this was originally added with "True".

Is this solution (using "False") going to work for MC as well as for data?

Reply:
@rekovic 
please comment
Thank you.